commands/load.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -euo pipefail
+
+arg=""
+
+if plugin_read_value_into_result BUILDKITE_PLUGIN_DOCKER_LOAD ; then
+  arg="${result[0]}"
+fi
+
+if [[ -z "$arg" ]]; then
+  echo "Error: load has to be given a path to the file that should be loaded"
+  exit 1
+fi
```

Comment:
According to the plugin spec, the `load` option is just a string so it makes a lot more sense to just do:

```suggestion
arg="${BUILDKITE_PLUGIN_DOCKER_LOAD:-}"

if [ -z "$arg" ] ; then
  echo "Error: load has to be given a path to the file that should be loaded"
  exit 1
fi
```